File: backend/RiskManagementAPI/Models/RiskRegister.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RiskManagementAPI.Models
{
    [Table("risk_register")]
    public class RiskRegister
    {
        [Column("id")] public int Id { get; set; }

        [Required] [Column("project_id")] public int ProjectId { get; set; }

        [Required]
        [StringLength(30, MinimumLength = 1)]
        [Column("name")]
        public string Name { get; set; }

        [Required] [Column("description")] public string Description { get; set; }
    }
}

File: frontend/riskManagementTool/src/app/model/riskRegister.ts
export class RiskRegister {

  public id: number;
  public projectId: number;
  public name: string;
  public description: string;
  public risks: object;

  constructor(id?: number, projectId?: number, name?: string, description?: string, risks?: object) {
    this.id = id;
    this.projectId = projectId;
    this.name = name;
    this.description = description;
    this.risks = risks;
  }
}